elvis.config

```diff
@@ -7,11 +7,6 @@
         filter => "*.erl",
         ruleset => erl_files
        },
-      #{dirs => ["."],
```

Comment:
`rebar3_lint` (now the most recent version) told me this wasn't being used... `rebar3_lint` is also killing it with fire! 😄